app/ts-monitor/run/handler_metrics.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 openGemini Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package run
+
+import (
+	"net/http"
+
+	"github.com/openGemini/openGemini/app/ts-monitor/collector"
+	"github.com/openGemini/openGemini/lib/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	metricMsts = []string{"httpd", "performance", "io", "executor", "system", "runtime",
```

Comment:
Some data tables is meaningless, advised to remove them, such as `sql_slow_queries`.